app/views/requests/_request.html.erb

```diff
@@ -72,6 +73,10 @@
         <dt class="col-3 offset-1 col-md-2 offset-md-2">Requested:</dt>
         <dd class="col-8"><%= l(request.placed_date, format: :short) %></dd>
       <% end %>
+      <% if request.proxy_requester_name %>
```

Comment:
Isn't this the same information that we're showing in the "Borrower" `dt` above for Symphony?

Reply:
That <dt> is nested under group, which I thought we were getting rid of before today's conversation. I will work on this now that I understand groups better.